<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="3" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-02T20:09:24 BST" name="Suite" finished-at="2022-06-02T20:12:54 BST" duration-ms="209797">
    <groups>
    </groups>
    <test started-at="2022-06-02T20:09:24 BST" name="Test" finished-at="2022-06-02T20:12:54 BST" duration-ms="209797">
      <class name="tests.StoreTests">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:09:24 BST" name="beforeSuite" finished-at="2022-06-02T20:09:24 BST" duration-ms="24" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:09:24 BST" name="beforeTest" finished-at="2022-06-02T20:09:24 BST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:09:24 BST" name="beforeClass" finished-at="2022-06-02T20:09:30 BST" duration-ms="6145" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:09:30 BST" name="beforeMethod" finished-at="2022-06-02T20:09:30 BST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyRegisterUser()[pri:1, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:09:31 BST" name="verifyRegisterUser" finished-at="2022-06-02T20:09:55 BST" duration-ms="24482" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRegisterUser -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:09:55 BST" name="afterMethod" finished-at="2022-06-02T20:09:55 BST" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method signature="verifyReRegisterUser()[pri:2, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:09:55 BST" name="verifyReRegisterUser" finished-at="2022-06-02T20:10:26 BST" duration-ms="30927" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:643)
at tests.StoreTests.verifyReRegisterUser(StoreTests.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyReRegisterUser -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:09:55 BST" name="beforeMethod" finished-at="2022-06-02T20:09:55 BST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:10:26 BST" name="afterMethod" finished-at="2022-06-02T20:10:26 BST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:10:26 BST" name="beforeMethod" finished-at="2022-06-02T20:10:26 BST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyLogin()[pri:3, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:10:26 BST" name="verifyLogin" finished-at="2022-06-02T20:10:42 BST" duration-ms="16068" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:10:42 BST" name="afterMethod" finished-at="2022-06-02T20:10:42 BST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:10:42 BST" name="beforeMethod" finished-at="2022-06-02T20:10:42 BST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyInvalidLogin()[pri:4, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:10:42 BST" name="verifyInvalidLogin" finished-at="2022-06-02T20:10:51 BST" duration-ms="8559" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [http://localhost/Avactis/home.php] but found [http://localhost/Avactis/sign-in.php?timestamp=1654197047]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [http://localhost/Avactis/home.php] but found [http://localhost/Avactis/sign-in.php?timestamp=1654197047]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at pageobjects.MyAccountPage.isLoaded(MyAccountPage.java:44)
at org.openqa.selenium.support.ui.LoadableComponent.get(LoadableComponent.java:52)
at pageobjects.MyAccountPage.<init>(MyAccountPage.java:32)
at pageobjects.SignInPage.login(SignInPage.java:82)
at tests.StoreTests.verifyInvalidLogin(StoreTests.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyInvalidLogin -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:10:51 BST" name="afterMethod" finished-at="2022-06-02T20:10:51 BST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:10:51 BST" name="beforeMethod" finished-at="2022-06-02T20:10:51 BST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyProductsAddedToCart()[pri:5, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:10:51 BST" name="verifyProductsAddedToCart" finished-at="2022-06-02T20:12:12 BST" duration-ms="80990" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyProductsAddedToCart -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:12:12 BST" name="afterMethod" finished-at="2022-06-02T20:12:12 BST" duration-ms="9" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:12:12 BST" name="afterClass" finished-at="2022-06-02T20:12:54 BST" duration-ms="41663" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:12:54 BST" name="afterTest" finished-at="2022-06-02T20:12:54 BST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:tests.StoreTests@2257fadf]" started-at="2022-06-02T20:12:54 BST" name="afterSuite" finished-at="2022-06-02T20:12:54 BST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- tests.StoreTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
